common/common.go

```diff
@@ -87,3 +93,31 @@ func SwapNibbles(k []byte) []byte {
 	}
 	return result
 }
+
+// ToTOML encodes a state type into a TOML file.
+func ToTOML(file string, s *cfg.Config) *os.File {
```

Comment:
This explicitly encodes a `cfg.Config` which makes me believe it belongs in the `cfg` package (maybe as a receiver function on `cfg.Config`?). You might consider abstracting this to take an interface or something of those thoughts. Lmk what your thoughts are

Reply:
Moved to `cfg` package, since it's only used in a unit test right now I do not think it needs to be a receiver function or an interface. Should this be used elsewhere within the source code we can consider taking those steps.